src/screens/videos/desktop.js

```diff
@@ -3,69 +3,75 @@ import { VerticalScrollbar } from "./vertical-scrollbar";
-          <h1>Videos you might like</h1>
-        </div>
+    <>
+      <BackgroundEffect type="videos-screen" />
 
-        <div className="mx-8 grid h-full grid-cols-12 gap-8 overflow-hidden p-2 xl:mx-32 2xl:mx-96">
-          <div className="col-span-9 h-full">
-            {videos.map((video) => (
-              <YoutubeVideo
-                key={video.id}
-                videoId={video.id}
-                className={cx("h-4/5", video.id !== playingVideoId && "hidden")}
-                onReady={onReady}
-              />
-            ))}
+      <section
+        id="videos"
+        className={cx("videos relative min-h-screen flex-col py-24", className)}
+      >
+        <div className="relative mx-auto flex w-full max-w-7xl grow flex-col space-y-10 px-8">
+          <h2 className="text-center font-brand text-4xl text-white">
+            Videos you might like
+          </h2>
 
-            {playingVideoId === null ? (
-              <VideoPreview
-                video={videos[0]}
-                className="h-4/5 rounded-t-xl p-5"
-                playBtnClassName=""
-                playBtnWrapperClassName="flex grow items-center justify-center mt-12"
-                titleWrapperClassName="px-6 py-3"
-                authorClassName="text-lg"
-                titleClassName="text-xl uppercase"
-                onClick={onPlay}
-              />
-            ) : null}
+          <div className="grid grow grid-cols-12 gap-8 overflow-hidden p-2">
+            <div className="col-span-8 flex flex-col xl:col-span-9">
+              {playingVideoId !== null ? (
+                <YoutubeVideo
+                  videoId={playingVideoId}
+                  className="grow"
+                  onReady={onPlayerReady}
+                />
+              ) : (
+                <VideoPreview
+                  video={videos[0]}
+                  className="grow rounded-t-xl p-5"
+                  playBtnWrapperClassName="flex grow items-center justify-center"
+                  titleWrapperClassName="px-6 py-3"
+                  authorClassName="text-lg"
+                  titleClassName="text-xl uppercase"
+                  onClick={onPlay}
+                />
+              )}
 
-            <div className="h-1/5 rounded-b-xl bg-khaki p-4 text-lg text-black shadow-[0px_0px_3px_1px_rgba(255,255,255,1)]">
-              Wild animal suffering is about harms to the wellbeing of animals
-              living in the wild. This is frequently confused with the ways
-              species, populations of animals, or ecosystems can be affected.
-              These are entities, not individuals who can feel and suffer. In
-              this video, you&apos;ll hear about the meanings of &quot;wild
-              animal suffering&quot; and related terms.
+              <div className="rounded-b-xl bg-khaki p-4 text-lg shadow-[0px_0px_3px_1px_rgba(255,255,255,1)]">
```

Comment:
We can define a shadow in the Tailwind config and use that instead of these arbitrary values.